openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/bike/CommonBikeFlagEncoder.java

```diff
@@ -700,13 +700,13 @@ void collect(ReaderWay way, double wayTypeSpeed, TreeMap<Double, Integer> weight
         }
 
         // MARQ24 MOD START
```

Comment:
Is the `// MARQ24 MOD` comment still valid after the update?

Reply:
sorry - done